Fixes #410 Updated NSEIndia stock price url as well as updated column names

maruhgar opened the pull request:
NSEIndia quote fetching broke after the url changed on 8th July.

Updated the url as per [this link](https://tradingqna.com/t/has-nse-changed-bhavcopy-location/169551).  Besides the url, the column names have also changed.  Updated this as well.
Ran the online test for NSEIndia and confirmed all tests pass.

bpschuck commented:
Thank you for contribution. @bgr22112 noted that the BSEIndia.pm module requires the same fix. Additionally the maintainers have been requesting that PRs also have the appropriate updates to the `Modules-README.yml` and `Changes` files.

maruhgar commented:
> Thank you for contribution. @bgr22112 noted that the BSEIndia.pm module requires the same fix. Additionally the maintainers have been requesting that PRs also have the appropriate updates to the `Modules-README.yml` and `Changes` files.

Since I don't use BSEIndia and since the original bug was for NSEIndia, I had omitted changes to it.  Fixed BSEIndia.pm as well and updated the Modules and Changes file.  Thanks!

bpschuck commented:
Thanks again, but PR #419 is going to be used since @bgr22112 was the also the last to update these two modules.
